### ISSUE TITLE
About Norm and BatchSize

### ISSUE THREAD

**Issue by tangjilin** [junyanz/pytorch-CycleGAN-and-pix2pix#342]
If I want to use the instance norm, can I set the batch size > 1?  In other word, if batch size>1, does it mean the norm way that I use  is always batch norm rather than instance norm? Thanks!

**Comment by taesung89** [comment]
"InstanceNorm with batch size > 1" and "BatchNorm with batch size > 1" are different things. Instance norm computes activation statistics per sample, while batch norm computes it over all samples in the minibatch. "InstanceNorm with batch size > 1" and "InstanceNorm with batch size == 1" are also different, because in the former case, we average the gradients from samples. 

You can try any of these in this code repo. They tend to produce different results. The default setting is Instance Norm with batch size == 1.
